### DISCUSSION TITLE
Latitude 3450 support?

### DISCUSSION THREAD

**Discussion by mrbeats09** [SkyrilHD/Dell-Exx50-Hackintosh#49] [Q&A]
Just wondering, but can the latitude 3450 also run using this EFI? 
It's basically just a 14-inch version of the 15-inch 3550. 

My model has a Haswell processor, though.

**Comment by mrbeats09** [comment]
Nevermind, I tried using it and it did not work. 
Gets stuck at the very first line (in verbose). 
It might work for the Broadwell 3450 models, though.

**Comment by SkyrilHD** [comment]
In theory it should work, however since you run a Haswell CPU you need to adjust some quirks, dortania's guide is amazing at explaining stuff so follow this: https://dortania.github.io/OpenCore-Install-Guide/config-laptop.plist/haswell.html

If you are still stuck, check the troubleshooting guide: https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/troubleshooting.html

**Reply by tharindupeiris0425** [reply]
Worked Perfectly On Dell Latitude 3450 but Need to install itlwm + Heliport to Wifi Work
Lap Specs :-
Intel Core i5-5200U
Hynix 8GB Ram
Intel HD 5500
addlink 256GB SSD
